Cargo.lock

```diff
@@ -3595,7 +3595,7 @@ dependencies = [
 [[package]]
 name = "webrender_api"
 version = "0.56.1"
-source = "git+https://github.com/servo/webrender#a21197eb1e427730c7bec6eec07bc6cc352d452e"
+source = "git+https://github.com/servo/webrender#007c61df33081caec1163db17305ccc191da7d99"
```

Comment:
I am wondering why we just need to change `source` and don't need to change `version`?
I thought when it update, it might from `0.56.1` to `0.56.2`?

Reply:
It's a git dependency, there's no need for versions.

Reply:
I am confused. So, when will the version change?

Reply:
It's updating the commit hash in which the package should be pulled and build from, so no version changes are necessary.

Reply:
If you're wondering about the version string itself, the Cargo.toml corresponding to that commit hash hasn't change its version (it's still 0.56.1), so there's no update.